support/values.yaml

```
@@ -71,3 +71,22 @@ grafana:
           access: proxy
           isDefault: true
           editable: false
+
+nfs-server-provisioner:
+  enabled: false
+  storageClass:
+    # Users will have to opt-in to using NFS-type storage under
+    # singleuser.storage.dynamic.storageClass
+    name: nfs
+    create: true
+    defaultClass: false
+    mountOptions:
+      - vers=4
```

```suggestion
      - vers=4.2
      - soft
      - noatime
```

From https://github.com/2i2c-org/pilot-hubs/blob/35a830bf5894f8a96244e265c9c9c0968eda490c/hub-templates/basehub/values.yaml#L22, which is our current set of options